src/lib/evil/evil_time.h

```diff
@@ -3,8 +3,11 @@
 
 #include <evil_api.h>
 
+#include <stdint.h>
 #include <time.h>
+#include <stdint.h>
```

Comment:
```suggestion
#include <stdint.h>
#include <time.h>
#include <stdint.h>
#include <stdint.h>
```

Y'know, just to make sure.